.github/workflows/ci.yml

```
@@ -60,6 +60,7 @@ jobs:
 
     env:
       IS_MASTER_BUILD: ${{ !matrix.numfocus_nightly && matrix.os == 'ubuntu-latest' && matrix.pyarrow == '0.17.1' && matrix.python == '3.8' }}
+      IS_TAG: ${{ github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')}}
```

nitpick to make life simpler
```suggestion
      IS_TAG: ${{ github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')}}
      IS_RELEASE: ${{ env.IS_MASTER_BUILD == 'true' && env.IS_TAG == 'true }}
```

---

This doesn't work. You canny use `env` in the `env` section. I've tried that before :D